vita3k/gui/src/settings_dialog.cpp

```diff
@@ -922,6 +922,10 @@ void draw_settings_dialog(GuiState &gui, EmuEnvState &emuenv) {
         ImGui::Checkbox("Grid Mode", &emuenv.cfg.apps_list_grid);
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Check the box to set the app list to grid mode.");
+        ImGui::SameLine();
+        ImGui::Checkbox("Check for updates", &emuenv.cfg.check_for_updates);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Automatically check for updates at startup.");
```

Comment:
set it here have no sense, move it inside emulator tab

Reply:
done

Reply:
thx